Flandstation

PestoVerde322 opened the pull request:
## About The Pull Request

Welcome to Flandstation (Too late to change name now!)
![FlandStation19](https://user-images.githubusercontent.com/75247747/147258717-d58ae1eb-9f16-483c-b49d-b1a66a6d7231.png)

_map-wide screenshot, NOW in 8-bit png resolution_
_V1.8 screenshot_ : https://user-images.githubusercontent.com/75247747/144725533-0f4decf3-e3c1-4444-9f85-9555bb7209d1.png
_V1.7-1.7.1 screenshot_ : https://user-images.githubusercontent.com/75247747/143782090-be40f71b-73c0-4981-a3ba-855d2e0c50b6.png
_V1.6-1.6.1 screenshot_ : https://user-images.githubusercontent.com/75247747/142921743-81f887fd-b948-4436-b4a0-312dac8bfb0b.png
_V1.5-1.5.5.1 screenshot_ : https://user-images.githubusercontent.com/75247747/142438496-ef35485f-1c29-4fdf-9c8d-482a04c17f1d.png
_V1.4-1.4.2.1 screenshot_ : https://user-images.githubusercontent.com/75247747/140964802-ab561261-d708-4b85-8473-517d022a2545.png
_V1.3-1.3.1 screenshot_ : https://user-images.githubusercontent.com/75247747/140725088-c0aa3e24-fcdd-42cf-88f3-5b5e13ca19fb.png
_V1.2.2-1.2.4 screenshot_ : https://user-images.githubusercontent.com/75247747/139871055-0e9349ea-9e51-4adc-b71e-30083881d874.png
_V1.2.1 screenshot_ : https://user-images.githubusercontent.com/75247747/139529852-ae787f2a-5f92-4fe8-a829-10b731f2822d.png
_V1.2.1 screenshot_ : https://user-images.githubusercontent.com/75247747/139422318-78b1ac5a-ee0d-438b-bfe9-4bc580409d0e.png
_V1.2 screenshot_ : https://user-images.githubusercontent.com/75247747/139331846-838a58df-a8d3-44b9-8d96-f3ed5ae178ee.png
_V1.1 screenshot_ : https://user-images.githubusercontent.com/75247747/138764896-b69f191d-a464-4dcf-80de-450c2f0ffe87.png
_V1 screenshot_ : https://user-images.githubusercontent.com/75247747/138606240-f2b074fb-7dc1-454e-987e-b5669a9d8bf7.png

_Forum thread :_ https://forums.beestation13.com/t/flandstation-feedback-thread/15539

Flandstation is a brand new and completely original station with several unusual design choices (and to be tested still).
The map contains an unusual spread out structure and amped up areas, alongside some non conventional stuff, like the perma-asteroid or the security being in the middle of the station.

Flandstation lore (personal headcanon):
As a former Syndicate outpost, that was hastly abbandoned for unknown reasons, this station sits in an asteroid cluster that hid the station quite well from pesky eyes. 
Nanotransen saw a great economical opportunity to build over the former outpost (originally just the current security and engineering).After a year of planning and construction, including a direct connection to Centcomm, the station currently house one of the finest research and developement labs, alongside future plans to make the station a major touristic and commercial hub.
What Nanotransen don't know, is that the Syndicate want the place back with the flooding reports of the pricy resource known as plasma being very prominent in the area, and it's hell bent to do everything to either get it back or destroy it to hide whatever they where hiding there (even if Official and confidential Nanotransen reports state that the station bareley had anything into it at the time of discovery).

## In-Game screenshots
![Engineering](https://user-images.githubusercontent.com/75247747/138698812-a19eb496-5d88-4ba7-874d-2876f5fcb532.png)
![Bar](https://user-images.githubusercontent.com/75247747/138698817-9b99545c-af8e-417f-b286-2630fb0f388e.png)
![Garden](https://user-images.githubusercontent.com/75247747/138698820-23e8b5be-5840-4875-8890-dedeabe3bc7d.png)
![Medbayentrance](https://user-images.githubusercontent.com/75247747/138698824-b0f82e1a-dbcc-47ef-a957-f247a15b4ffe.png)
![Arrivalfoyer](https://user-images.githubusercontent.com/75247747/138698827-d225c393-57ce-4371-bff5-d63d8cc249b1.png)
![Permafoyer](https://user-images.githubusercontent.com/75247747/138698832-6b4c93fe-1617-49ec-978f-3b6c65a430a3.png)
![twinturbo](https://user-images.githubusercontent.com/75247747/138698836-976f80d9-b0f3-4379-837d-fbb8a1517ab3.png)
![Bridge](https://user-images.githubusercontent.com/75247747/138698840-57f93ff6-a9d9-45b6-9cda-28d10d71b99e.png)

## Why It's Good For The Game

It adds a new map, that's all really. Oh and also Atmos decal and a new carpet.

## Changelog
:cl:
add: Flandstation
add: Shuttles (Whiteship, arrival, cargo, emergency, exploration, ferry, labour, mining).
add: A new type of carpet, the Grimy_carpet.
add: New markings for the Atmos.
add: New marking for directions on the floor.
add: New (map dedicated) type of random room.
code: Modified the code related to the markings.dm to allocate the new markings for atmos.
code: Modified the code for fancy_floor.dm to allocate the new carpet type.
code: Modified the code for Laborshuttle.dm for the new Labor shuttle destinations.
/:cl:

PestoVerde322 committed:
Flandstation

A new station for Beestation

Crossedfall reviewed (dismissed):
The map needs to be saved in the TGM format. Open it with something like [StrongDMM](https://github.com/SpaiR/StrongDMM) and then save it again, that should convert the file for you without having to futz with the script.

PestoVerde322 committed:
Fixed Shuttle files format

From BYOND to TGM

PestoVerde322 committed:
Removed emergency_sorry

Joke shuttle that wasn't meant to get in the pull in the first place, whoops :V

Archanial commented:
Did you actually make it in byonds default editor?

AnCopper commented:
Your escape pod is missing a wall.
![image](https://user-images.githubusercontent.com/82319489/138718404-788de602-438a-4e3f-a87e-71736fd6f883.png)

PestoVerde322 commented:
> 
> 
> Did you actually make it in byonds default editor?

nope, i used the wrong settings in StrongDMM for a loooong while before setting it to the right format

PestoVerde322 commented:
> 
> 
> Your escape pod is missing a wall. ![image](https://user-images.githubusercontent.com/82319489/138718404-788de602-438a-4e3f-a87e-71736fd6f883.png)

I'm not sure why the game decided to not render that turf, but on my end it is fine

![dreamseeker_OtO7Io5RKC](https://user-images.githubusercontent.com/75247747/138719290-c26a7fbb-67a1-4f27-8294-50a1b068bd91.jpg)

Kapu1178 commented:
HOLY SHIT A NEW STATION

ike709 commented:
🎉

Archanial commented on code/game/objects/items/stacks/tiles/tile_types.dm:
```diff
@@ -255,6 +255,16 @@
+	item_state = "tile-carpet-grimy"
+	merge_type = /obj/item/stack/tile/carpet/grimy
+	turf_type = /turf/open/floor/carpet/grimy
+	//tableVariant = /obj/structure/table/wood/fancy/grimy
```
commented code

Archanial commented on code/game/area/Space_Station_13_areas.dm:
```diff
@@ -77,6 +77,28 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
+
+/area/docking
+	ambience_index = AMBIENCE_MAINT
+	//valid_territory = FALSE
```
comments

Archanial replied:
more commented valid territory bellow

Archanial reviewed (changes requested):
After pulling and compiling your branch on local you get undefined object old_taser

francinum commented:
![](https://file.house/TIKn.png)
This doesn't look good, and probably doesn't work.

francinum commented:
You forgot to clean up some swap space near the chapel
![image](https://user-images.githubusercontent.com/5572280/138742001-e3052794-ff5f-4f0a-bd63-cdf32d60caee.png)

francinum commented:
Why does it appear that most heads of staff are given energy pistols?

Archanial commented:
I have only looked at some parts of the map

![image](https://user-images.githubusercontent.com/37456678/138736147-842af157-e522-437f-b843-0143757ed6ab.png)
not connected
![image](https://user-images.githubusercontent.com/37456678/138736472-2fb13dfd-63b1-4469-99bc-42384433aa96.png)
I personally don't like putting glass next to grille
![image](https://user-images.githubusercontent.com/37456678/138736624-13b5a25f-b9b7-4f20-901a-4d20738682d9.png)
this doesn't look right

![image](https://user-images.githubusercontent.com/37456678/138737472-b8b6b73f-9b23-40da-b14d-bd9a9d622969.png)
no
![image](https://user-images.githubusercontent.com/37456678/138737517-501a5af4-51da-416a-bd58-a9a4361ffec9.png)
stock computer bad, I suggest deleting it and moving plant here

![image](https://user-images.githubusercontent.com/37456678/138737885-55e9400e-4ac9-45bb-82d6-07a20abf4f2a.png)

Weird design choice, why did you make arrivals have it's own atmos input?

![image](https://user-images.githubusercontent.com/37456678/138738042-5faddeaa-8471-410c-8461-b01fe516ec62.png)

Probably not intended

![image](https://user-images.githubusercontent.com/37456678/138738258-3ddef435-d025-4594-b17a-b02556fd6ca5.png)

default computer

![image](https://user-images.githubusercontent.com/37456678/138738561-ab683b35-e81d-4663-ab3f-c56c4a6b297c.png)
as much as I like the idea, I don't think it's a good idea to put a vtec module here

![image](https://user-images.githubusercontent.com/37456678/138738953-ddee46d2-f01f-455f-a3b0-7ab5a4ed0f2f.png)
not connected

![image](https://user-images.githubusercontent.com/37456678/138740177-899d2bbe-7f9d-43e5-9a68-ff8f79ddd8fb.png)
this room has no vents or scrubbers

![image](https://user-images.githubusercontent.com/37456678/138740447-494b572d-ab04-4506-b841-262be34c14e8.png)

duplicate vending machines

![image](https://user-images.githubusercontent.com/37456678/138740761-8fe86e5e-50b2-4e75-bf93-67c155302113.png)
messed pipes

![image](https://user-images.githubusercontent.com/37456678/138740898-0303e05d-7d55-4208-97bb-9eb16cb5bcee.png)

not connected (north solars)

![image](https://user-images.githubusercontent.com/37456678/138741046-4335a2ab-f06a-4f7b-91ee-0dd7557265bc.png)

I suggest going under the door instead

![image](https://user-images.githubusercontent.com/37456678/138742112-17b5ce27-6acf-42d6-9460-2614ed7ae6d0.png)
No map needs 3 sleepers and 4 stasis beds, 2 beds and 1 sleeper is enough, rest should be left for engineers to construct

![image](https://user-images.githubusercontent.com/37456678/138742426-e932ecc5-7ea1-4315-bf5b-9e2cacfda9f4.png)

freezer is missing vent

![image](https://user-images.githubusercontent.com/37456678/138742530-d72fa585-c5ae-453e-ac55-05df529dea9e.png)

I very much prefer having your locker inside of the cell, and they feel too small (maybe on purpose)

![image](https://user-images.githubusercontent.com/37456678/138742886-8cf18a6f-2f68-4456-b883-763747dd55f5.png)

This perma feels small and unlike other permas is unescapable. 


![image](https://user-images.githubusercontent.com/37456678/138743068-0aa5b033-5f54-4aa0-8b76-5c3188c1a36e.png)

cable loop

![image](https://user-images.githubusercontent.com/37456678/138743108-99ba9338-732c-4e5e-a3ae-35db4e598d2c.png)

You can't get to the corner and slot machine feels out of place

![image](https://user-images.githubusercontent.com/37456678/138743235-8a9e88e2-53e8-4cc1-8a26-cce389897396.png)

this window has no purpose

![image](https://user-images.githubusercontent.com/37456678/138743341-9ffeff9f-208b-445e-aafd-5d22107db068.png)

imo it's too well protected, most of the station have an entrance from hallways
also QM is supposed to have an access to ore silo

![image](https://user-images.githubusercontent.com/37456678/138743519-bd7696f5-47f6-43b1-8a22-cfb06e210b67.png)

this cable doesn't connect (north part of bridge)

![image](https://user-images.githubusercontent.com/37456678/138743615-79ca89f5-06ad-4664-ba8b-ffeb8d623c40.png)

AI upload should be separate room

![image](https://user-images.githubusercontent.com/37456678/138743704-63a00cb9-7d06-45c8-bfbb-8f47c61ab6ae.png)

You can't get to AI without pickaxe, which is a very specific item to have

PowerfulBacon commented:
is there meant to be a taser in the warden's locker?

Ivniinvi commented:
![image](https://user-images.githubusercontent.com/22421329/138756463-09de1b3c-2dca-43a6-b943-1fc1cd8cb4d2.png)
This airlock on the shuttle really feels tacked on after the fact, there's more than enough tiny fan doors on the shuttle, I'd recommend removing the airlock entirely.

Also the pistol on the shuttle bridge might be a bit controversial.

PestoVerde322 committed:
Flandstation 1.1

First batch of fixes *see comment when this is pushed*

PestoVerde322 commented:
Aright, thanks everyone for your comments and support, with the aid of the official Beestation discord too.

Here's the first batch of fixes and changes (with the user who pointed out the various bugs):
-Removed the superfluous comments in tile_types.dm and Space_Station_13_areas.dm (_Archainal_)
-Updated Disposal to make the grinder to work properly (_Francinum_)
-Removed misc leftovers around the station (_Francinum and Archainal_)
-Removed the leftover Energy pistols, PTSD and the Battered tazers scattered trought the station and the Energy pistol in the Emergency Shuttle (_Francinum, Archainal, PowerfulBacon and Ivniinvi_)
-Added and modified odd piping choices (_Archainal_)
-Fixed the Default computers trought the station (_Archainal_)
-Removed the VTEC (**kicks in yo**) (_Archainal_)
-Added vents and scrubber in the Detective Office (_Archainal_)
-Removed the Duped vending machines in the D&D room (_Archainal_)
-Removed the surplus of medical equipment in medbay (_Archainal_)
-Added the missing vent in the Cold room (_Archainal_)
-Added a big window to the leisure cell (_Francinum and Archainal_)
-Other misc changes that would make this list far too long...

Non addressed changes:
-Accessory air loop in arrival (kept there in case of sudden depressurization of arrival as a redundant system)
-AI upload not shifted in an side room (to be tested in a live game)
-AI chamber surrounded by stone (to be tested in a live game)
-Vault being too tightly secured (to be tested in a live game)
-Brig cells being small (Done in a similar vein as Kilostation)

ike709 commented:
Reminder to myself and the other codejannies that we need to bug affected to update the webmaps when this is merged.

williamcll commented:
I love it, it's like an upsized kilostation. and lots of building space too.

How does perma asteroid work?

PestoVerde322 commented:
> 
> 
> I love it, it's like an upsized kilostation. and lots of building space too.
> 
> How does perma asteroid work?

First off, the Brig shuttle (by default in the security dock) has multiple docking spots around the station (Arrival, Bridge, Permaroid and Brig), then it works as if you are going to the Lavaworld location, with (in theory) the mini perma being either a holding cell to crews who are going to get perma'd or as a long term VIP cell.

Tsar-Salat commented:
Where are external cameras, I see none? 

This looks especially an issue for external AI sat.

PestoVerde322 commented:
> 
> 
> Where are external cameras, I see none?
> 
> This looks especially an issue for external AI sat.

As for now, in the Current WIP version i have added external motion tracking cameras around the AI core, but if I should add more in more "critical" locations, let me know!
![StrongDMM_WrSeChcGC6](https://user-images.githubusercontent.com/75247747/138941461-74429a22-f85b-4ce2-8088-a1a860a195a1.jpg)

MNarath1 commented:
> 
> 
> > I love it, it's like an upsized kilostation. and lots of building space too.
> > How does perma asteroid work?
> 
> First off, the Brig shuttle (by default in the security dock) has multiple docking spots around the station (Arrival, Bridge, Permaroid and Brig), then it works as if you are going to the Lavaworld location, with (in theory) the mini perma being either a holding cell to crews who are going to get perma'd or as a long term VIP cell.

just to be sure does this work with current supercruise code?

PestoVerde322 committed:
Flandstation 1.2

Wide Updates and fixes time *see comments for changeloaf*

PestoVerde322 commented:
This has been a busy midweek, many changes all around and many, MANY fixes and changes.

Again thanks for the user on the Beestation discord (and also here), for let me know on what should've been fixed and whatnot:

From TheButcher:
-Added Motion track cameras around the AI core for AI safetey purposes

From Archanial:
-SMES by garden is gone, Realized it wasn't worth to keep two whole SMES there if it's connected by the station directly
-West solars SHOULD BE fixed finally.
-Mining locker beneath West solars is gone, replaced by a pickaxe, a shovel and an air tank.
-Toned down some overbearing decals around

From Ivniinvi:
-Rotated a plasma tank that was in a wrong rotation.
-Rotated a thermomachine tha was also in a wrong rotation.
-Fixed some manifolds and decals over fixtures.
-Some aesthetical changes around Xenobiology.
-Mining shuttle is now more squared up.
-Plenty of Runtimes fixed up.
-Fixed a mirror-shower overlaps that didn't look nice at all.
-Medbay's defib mounts gone for balance sake.
-An out of place T med closet gone forever.
-Botany no longer has some Bmats.
-Made the Kitchen's shutter button more intuitive.
-Removed a redundant shutter by the kitchen.
-Toned down the many Windows into maintenance.
-Fixed the Air and some fixtures on board of the whiteship Eden, however for now it still drift trough space.
-Fixed an overabbundance of Petes, pinpointers and Nuke disks.
-Sgt-At-Armsky finally in the Armory as it should be.
-Made the hand teles more visible.

From spOcky:
-Added more Comm units trought the station (still more to be added).
-Shutters in the experimentation room can now be open in both ways.
-Overlapped fixtures now are no longer overlapping.
-Made the containment pen actually viable for containment.
-Removed a newscaster in the middle of Xenobiology.
-Fixed some air pipes that wasn't unconnected.
-Modified the exploration shuttle to fit a table and an extra chair.

Also from Ivniinvi, Archanial and Francinum:
-Toned down decals in security.
-Removed all marking beacons on ALL the shuttles.
-Firelocks on maint access have been removed.
-Made the <STAND CLEAR> decals more visible.
-All layer adapters should now be properly hidden on turfs.
-Added an arrival timer cell, and also one in medbay.
-Removed Contrabband locker by the Checkpoint.
-Made arrival decals more consistent.
-Server room has been reduced.
-Added a janitor door to the hallways.
-Removed the retro floors from the arcade.
-Removed many, many Stock computers and computer frames around the station.
-;)
-Vault's "self destruct" tiles (the circuit one) now are green instead of red.
-Shifted some cameras in the Atmos gas storage chambers to make the decals more clear.
-Made the Cooler space pipes more accessible.
-Added a tank dispenser by the Supermatter engine.
-Removed the twin turbine setting due to enviromental issues.
-Wired up the Emitters.
-Removed all instances of Fastmost firelocks.
-Removed an extra YouTools vending machine
-Removed an emergency pod from Brig.
-Removed some unnecessary fixtures on board of the evac shuttle, including a medical vendor lodged inside a seat.
-HoP's line has now two flashers on each side of the queue.
-Removed some random power cables in grilles in space.
-Made arrival airlock public once again.
-Chapel's mass driver now fires in the right direction.
-Perma's atmos made more dependent to the station (still have it's spare SMES).
-Added extra doors in the Vacant commissaries.
-Added more fixtures in general.

Now for some homebrew fixes:
-Removed unnecessary railings.
-Standardized the displacement of all fixtures (aside when it was needed to be more "different").
-Fixed RD's shutters.
-Added extra fixtures.
-Fixed some Atmos mess.

_**From the massive lists of bugs I still haven't fixed them all, however I am currently working on fixing more of them.**_

also to reply to MNarath1:
> just to be sure does this work with current supercruise code?

I have tested all the docks and it seems to work well with the actual Supercruise code, it's a bit weird to make them work around, but it works.

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Update from the Main codebase.

PestoVerde322 commented:
Also as an extra thing, don't mind the spare windows in the middle of the garden, accidental leftover, whoops.

PestoVerde322 committed:
Flandstation 1.2.1

minor updates and and fixes *see comments for changeroll*

PestoVerde322 commented:
Aright, in the version pushed yesterday there was some odd stuff that was leftover by error so have this slimmer changelog:

-Added a museum branch and a new Comf zone in the Library.
-Tweaked Shuttles and shuttle docking areas, looking more slick and less than either bricks or don't have senseless corners.
-Now the Security area on the shuttle will no longer lead to a wall at Centcomm.
-Removed the windows in the middle of the garden.
-Made the brig shuttle's dock doors accessible to all heads.
-Added a pipe connecting Air to Mix in Atmospheric.
-Misc aesthetical changes around the station.
-Added an extra SMES to the SMES room due to the power failing station wide in less than 5 minutes.
-Added lockers to the fighting arena.
-Other stuff i forgot to do.

PestoVerde322 committed:
Flandstation 1.2.2

Even more small fixes *see comments for changechange*

PestoVerde322 commented:
Aright, new round of small fixes and changes:
-Removed Chemestry access to pretty much everyone who isn't a chemist or whoever has access to chemestry.
-Removed some insuls in the main tech storage in trade of some toolbelts.
-Added some lights in the server room.
-Connected the cold loop that was missing a pipe somewhere.
-Fixed a disposal junction that was wrongly placed (twice).
-Made some pipes in Xenobio hidden.
-Added some extra windows in the mini perma for see from the inside of security what's going on there.
-Removed a shutter in the middle of the RD's office.
-Added the signage in case some assistant get lost.
-The whiteship STILL drifts.

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Update to the current Beestation Version from master.

PestoVerde322 committed:
Flandstation 1.2.3

Even more small fixes *see comments for logchange*

PestoVerde322 commented:
Hey all, some more fixes related to the map:
-Adjusted the many, MANY decals trough the station to either fit or to not being weird.
-Added cameras in the Engineering maintenance as it has view to some critical areas (turbine, first access to the permarock and the engine cool loop.
-Fixed some oddities related to the Solars area (missing floors on the Starboard Solars, Rwalls that don't really belong in the Aft solars).
-Other stuff that i don't really recall because i have done them loosely around.

francinum commented:
![](https://file.house/lJxV.png)
![](https://file.house/0YFd.png)
Both of these need quotes.

PestoVerde322 committed:
FlandStation 1.2.4

Fixed some big whoopsies *see comments for changeling*

PestoVerde322 committed:
FlandStation 1.2.4.1

Even more fixes of whoops (damn you missing tags *see previous changelog*

Archanial commented on config/maps.txt:
```diff
@@ -49,6 +49,11 @@ map kilostation
+map flandstation
+	maxplayers 100
+	votable
+endmap
```
```suggestion
map flandstation
	maxplayers 100
	votable
endmap
```
you got indentation wrong

Crossedfall commented:
![image](https://user-images.githubusercontent.com/26130695/140652149-45bd6cd8-3522-4e25-b0d4-2fe8d2d31326.png)

The tiles here are far too flat and don't actually fit with the rest of the style. I'd strongly recommend replacing them with something else.

![image](https://user-images.githubusercontent.com/26130695/140652184-2879067f-7761-4323-97e9-54133a1c8515.png)
![image](https://user-images.githubusercontent.com/26130695/140652194-771a8c6c-9a98-414c-8582-6d3b06901789.png)


The text doesn't look great. Especially in areas where there's a lot of it. I don't really get the Russian text reference either, I'd say that if you _really_ want the text decals they should probably just be in English to cut down on how much text you actually have.

PestoVerde322 committed:
Flandstation 1.2.5

No big changes aside the mining shuttle being broken.

Effemu commented:
Chapel and library being a the edge of the map makes people never go there.
Detached perma makes prisoners commit suicide faster.
AI sat with only one entrance...
Tool storage near command and not engineering.
Cargo is way too small.
Grav gen is in a weird nonsensical place
Yeah, nah

Oh and one way railings by the windows of "indoor" perma need to go.

PestoVerde322 commented:
Aright today the first Testmerge of Flandstation happened, and as expected, some bugs and whatnot happened all around soon-ish i will issue some radical changes related to it to the 1.3 version of the map.

> 
> [image]
> 
> The tiles here are far too flat and don't actually fit with the rest of the style. I'd strongly recommend replacing them with something else.
> 
> [image]
> 
> The text doesn't look great. Especially in areas where there's a lot of it. I don't really get the Russian text reference either, I'd say that if you _really_ want the text decals they should probably just be in English to cut down on how much text you actually have.

To be honest, i made them that way as a early sprite thing, however I'm planning to refine them and make them more spray-ish and also more conforming to the hexagon tiles, regarding the Russian+English test is from *that* old as hell atmos guide, and I have seen a grand majority of people enjoying them even as is, thus I don't really see a need to change them "content" wise, however I'll see if i can make them more eye pleasing, otherwise, in the rework they go.

> 
> 
> Chapel and library being a the edge of the map makes people never go there. Detached perma makes prisoners commit suicide faster. AI sat with only one entrance... Tool storage near command and not engineering. Cargo is way too small. Grav gen is in a weird nonsensical place Yeah, nah

This was something that at first I was rather concerned about, the map was originally planned for a really really high cap (50+ people online up to many many more) which is why it's VERY big. however it's a hard compromise of aesthetics and compactness, which is why chapel and library is quite far away, relatively speaking (think of cog1 in Goonstation, where botany, the detective's and the chapel are really, REALLY low traffic areas aside for the botanists that works there and the few pod users).

AI sat is a valid concern... except it's not a sat (and also, see kilo's AI sat, which has only one entrance).

Tech storage (I think, not the tool one) can be reshuffled somewhere else, but i don't really see being where it's now as a grand issue.

The permaroid is also something that is big concern, since concept wise was a good idea, but i realize that it will not really work at the end of the day, since slapping it in the middle of the station would make it even more of a hassle to get around. A way to fix it would to just remove the dock from the brig to make room for a proper (albeit small sized) perma, but then, where i can put the dock (aside also having the whole of the perma being a massive security flaw since a bomb will ruin perma capabilities and also some hallways)?

Cargo is fine as is, considered is similar to the one existing like delta or meta (only with an odd shape compared to the typical box shaped one, and have you seen kilo's cargo?).

Also you mentioned the gravity generator being in a "nosensical" spot, however you didn't comment about the Teleporter-Gateway being squished inbetween the chapel and the Library?

> 
> Oh and one way railings by the windows of "indoor" perma need to go.

The easiest thing that can be dismantled on board of the station, it just need some cooperation from the outside, prisoners aren't supposed to have an easy lone wolf escape, but then again, see above.

williamcll commented:
> Chapel and library being a the edge of the map makes people never go there. Detached perma makes prisoners commit suicide faster. AI sat with only one entrance... Tool storage near command and not engineering. Cargo is way too small. Grav gen is in a weird nonsensical place Yeah, nah
> 
> Oh and one way railings by the windows of "indoor" perma need to go.

Detached perma is good. More dead players to join ghost roles. And a more peaceful station when there's none (plus easier to find griefers since no living antags)

PestoVerde322 committed:
Flandstation 1.3

First Batch of fixes and changes after the Testmerge *see log containing changes*

PestoVerde322 commented:
This is the first changelog after the first testmerge, which had some bugs but they where easily addressable.
Anyway here's the changelog:
-Added extra benos spawn (aside the vents to do their xeno things).
-Added some loose copper around in the maints.
-Reduced the overabbundance of plasma sheets in the Apothecary.
-The box ring no longer doubles up as a tanning saloon.
-Due to Nanotransen's concerns, the Firelocks in the maintenance have been removed.
-Removed many glass windows surrounding computers.
-Reshuffled some maint lights around.
-Moved one of the evaquation pods to the library.
-Reshaped T-comms to make room for the Exploration comms system.
-Granted access to the Brig physician to his workplace.
-Miners should be able to leave cargo not only from the maint access.
-Reshaped a bit of chemestry and morgue to make more room out of them.
-Tweaked the many many lights all around to be more uniform and standardized.
-The evac shuttle no longer collide with the guide gliders and also, now properly docks with the station.
-Added some extra Medbay singage, but I'm not sure if it's going to be enough.
-Other miscelaneous changes that I forgot to be honest.

Also from the previous version (which i didn't made the changelog for it):
-The quickiest mining shuttle fix in the west.
-That was it really.

Also i Strongly Suggest to check the Feedback thread to also report bugs there: https://forums.beestation13.com/t/flandstation-feedback-thread/15539

PestoVerde322 committed:
Flandstation 1.3.1

Other Batch of (small) fixes and changes after the Testmerge *Observe the chart of changes*

PestoVerde322 commented:
Small fix ups that GrapeSmallseed on the forum pointed out and (indirectly) 
AnCopper on the Github, and also Francium pointing out a thing in game.

-Fixed the turbine not getting power due to not being connected at all (due to the dual turbine leftovers).
-Added AACs (Advanced Airlock Controls) to the auxiliary docking port.
-Made the Ferry shuttle less of a brick.

PestoVerde322 committed:
Flandstation 1.4

Structural updates and some fixups *See changelog*

PestoVerde322 commented:
Hey everyone, V1.4 is out, here's the changelog:

-Reshaped the chapel-library-teleporter asteroid, this should help a little with navigability alongside being a little more simpler to get around.
-Reshaped the Crew Quarters' dormitories, now they no longer have a massive hole in either sides.
-Speaking of Crew Quarters, the navigation for bots has been fixed up there, now they shouldn't get stuck anymore.
-Toned down decals stationwide, mostly engineering's checkers-like floor.

This version however's gonna be probably shortlived, as I forgot a couple of changes around, so stay tuned!

PestoVerde322 committed:
Flandstation 1.4.1

Extra fixups on 1.4 *See changelog*

PestoVerde322 commented:
And as expected, i fixed some broken stuff around:
-Fixed some odd cables and pipes being where they didn't belong.
-Added some extra intercomms,
-Added an extra light in the library's Evac Pod's dock.

Some other stuff hasn't been fully fixed it, like the chef's freezer's alarm going on full alert for detecting a temperature that's too cold and the drifting Whiteship.

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Monthly-ish update to master codebase

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation

Hopefully this won't blow up the whole project

PestoVerde322 committed:
Flandstation 1.4.2 / Experimental

Decal merging and some more fixups, very experimental, might end up very good or very bad *See the log*

PestoVerde322 commented:
Aright, i have fixed some stuff, and while the compact-er version of Flandstation I'm still working on (aka the "sugar-free" version), for now i have done some changes with an... experimental twist:
-Added the (experimental) merged decals, this should reduce the load on the map by a fair bit, but i need a definitive test in-live to see if they actually do something. This change will take in account of all the maps related to Flandstation (the station itself and all the shuttles), the other decal-based stuff is still the same.
-Added extra copper around the station.
-Fixed some decal oddities that either didn't made sense or they where ugly to the eyes.

Also structural wise, nothing changed, so for now the travel times are still massive for now.

Stay tuned for more info!

PestoVerde322 committed:
Flandstation 1.4.2.1 / Experimental

Fixed some oopsie related to Engineering *No changelog needed*

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Update to the current Beestation Version from master.

PestoVerde322 committed:
Merge remote-tracking branch 'origin/FlandStation' into FlandStation

PestoVerde322 committed:
Flandstation 1.5 / Experimental

Medbay update and stationwide fixes and restructuring *See changelog*

PestoVerde322 commented:
New update issued for flandstation (despite no testmerges happened)

I'm glad to present you the renewed medbay!
![FlandStation15med](https://user-images.githubusercontent.com/75247747/142437573-846c7c0c-0e29-40a2-93c4-da24e76a9507.png)

Now it should be easier to navigate and also, it features a brand new chemical hall for those chemist who like to have a massive network trough all of medbay.

Anyway here's the changelog:
-Renewed medbay to waste less usable space and also have more features clustered in.
-Renewed the kitchen to make room for a now easier to access service room.
-Added some new lights around the station, aswell some new cameras.
-Tweaked more decals with the Merged decals.
-Other stuff i've sureley forgot.

Stay tuned for more info!

PestoVerde322 committed:
Flandstation 1.5.1

Stationwide fixes *I'll do the changelog as soon as i can i swear!*

PestoVerde322 commented:
After an unnerving git corruption, i have done the needed quick fixes and uploaded the new version:
-Added some apologies letters to the Detective.
-Removed some old maintenance leftovers.
-Tweaked some broken pipes.
-Rewired some disconnected APCs.
-Finally added the spawns for the Exploration crew.
-Removed all the firelocks in maint.
-Other stuff i forgor 💀.

Stay tuned for further changes!

PestoVerde322 committed:
Flandstation 1.5.2

Tweaked some access and added some doors in security, added a proper distro-to-waste connector *no changelog needed*

PestoVerde322 committed:
Flandstation 1.5.3

Tweaked MORE stuff based on users feedback *changelog will come with a big delay*

PestoVerde322 committed:
Flandstation 1.5.4

Tweaked EVEN MORE stuff based on users feedback, AND REMEMBERED TO PUSH ALSO THE EXPLORATION SHUTTLE *changelog will be packed with the 1.5.3 version also*

PestoVerde322 commented:
Ok, I got caught up with tons of stuff yesterday that i forgot to issue the changelog for 1.5.3 and 1.5.4, so instead i am gonna put the changelogs together:
-Addressed the many Atmos related oddities pointed by Megaddd on the forum.
-Moved the E.X.P.E.R.I.M.E.N.T.O.R. console within the chamber for the time being, due to the console not connecting with the machine itself.
-Fixed the Morgue and Surgery's power issues.
-Moved the main surgery APC closer to the "active" area.
-Made morgue bigger (and gave it proper Air).
-The Exploration Shuttle Engine Heaters are covered now with normal glass to facilitate upkeep and upgrading.
-Added a Distro-to-Waste connector.
-Made security more secure against wandering non-authorized personnel.
-Added a "NO ERP" poster next to the Holodeck to remind people.
-Fixed unproper access around the station.
-Again, other stuff i forgot.

Also note to the maintainers, please upload the latest version to the server (8168890, aka the latest one) instead of the 1.5 / Experimental (7fb9cfb) as that version now is very outdated.

Stay tuned for further informations.

PestoVerde322 committed:
Flandstation 1.5.5

Renewed Virology, added some materials to Science, added some extra privacy to the VIP jail *see changelog for extra details*

PestoVerde322 commented:
Hey everyone, V1.5.5 has been pushed, here's the list of changes:
-Reworked Virology to have actual proper fixtures.
-Slightly reworked R&D and Xenobiology, finally no more complains about the lack of copper.
-Fixed Even more odd piping.
-Added some privacy to the VIP jail.
-Other small stuff I have surely forgot.

PestoVerde322 committed:
Flandstation 1.5.5.1

SPEEDFIX because there was an hole in the medical maintenance plus some extra small changes *See changelog*

PestoVerde322 commented:
SPEEDfix because I actually forgot an empty tile beneath viro, depressurizing all of the medical maintenance, also tweaked a little more Virology and the teleporter.

PestoVerde322 committed:
Flandstation 1.6

Moved the tech storage around, add a (temporary) construction room *See the logchange*

PestoVerde322 commented:
Aright, after further feedback, V1.6 is ready!

Changes:
-Moved the Tech storage right in front of engineering to quell the engineering complains of having to go down all the entire hallway to get some boards.
-In trade, where it used to be the Tech storage, now there's a construction site with a quite unusual window into it and the food commissary.
-The Auxiliary tool storage has been moved above the Art storage.
-A ton of fix related to odd lighting, broken piping, and general oddities has been patched out.
-Other minor stuff that has been fixed (thanks discord and forum!)
-Slightly modified the dock in the lavaland to accommodate the brig shuttle, don't worry, it will not break the other shuttle's docking.
-The Labour shuttle has been changed to fit a more normal layout.

Stay tuned for more changes!

PestoVerde322 committed:
Flandstation 1.6.1

Added some extra disposals and nicotine dispensers, made the map not suitable for lowpop *See the list of changes*

PestoVerde322 commented:
Aright update V1.6.1 is out based on some more user feedback:

-Added extra disposal units trough the station to the cleanliness maniacs out there.
-Added extra Nicotine vendors, especially next to medbay.
-Made the bar less depressing and more bright.
-Added a camera in front of the Gravity generator.
-Removed grilles to the Starboard Solar array.
-Fixed more decal stuff.

I have also tweaked the maps.txt file to made it so that the map won't be available if the user population is below 45 users, meaning that now it's more of an highpop map than a lowpop one.

Stay tuned for more changes.

SergeiKoralev commented:
Don't know if it's been said yet, but arrivals overpressurizes like... every round.
The outer asteroid prison wing air alarm is placed in a room that starts decompressed, causing an air alarm at round start.
There is an air alarm in a door just north of the library.
Monastary maint APC is not connected to the grid. (the cable is cut just outside the airlock)
The aft two airlocks on the escape shuttle don't line up with the docking ports.

SpinnerMaster commented:
Cargo is a little hard to work in as you cannot tell if the shuttle is docked without checking the console. I think a nice change would be to make some of these walls windows instead: 

https://i.imgur.com/S1oGVm4.png

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Update on the branch project thing that does thing about update do i really have to comment this?

PestoVerde322 committed:
Flandstation 1.7

New Exploration shuttle, First iteration of the front random room, Stationwide fixes and adjustments *Spot the log change*

PestoVerde322 commented:
Aright, time to update the map once more and see where this goes, this one has big changes all around:

-Added a new Exploration shuttle, courtesy of Spockyeyeye!
-Adjusted the fore solar panels to fit the new Exploration shuttle.
-Moved the corporate showcase to where there was the rock garden by Science/engineering
-Added a new random room to where there was the corporate showcase, so far it has 8 random rooms with different themes and details.
-Fixed the many, many bugs around the station (wrong piping, odd decals, that disposal pipe in the wall, etc...)
-Adjusted some Atmospherics piping.
-Fixed the ORM ejecting the ores in the wrong direction.
-moved the Mulebots parking to make room for putting crates on the conveyor belt system to disposal.
-Standardized the light switches and the fire extinguisher closets around the station.
-Defined and adjusted the VIP cells zone to what they should be, VIP cells, *REMEMBER* that there's a proper perma south east of the station a brig shuttle trip away!
-Added new buttons for the shutters of the VIP cells
-Switched the location of the botanical chem dispenser and the plant trait disk storage toaster based on what other stations do.
-Added an extra Nanomed Plus in medbay.

Stay tuned for more update logs!

PestoVerde322 committed:
Flandstation 1.7.1

Fixed an oopsie with the emergency shuttle, added an extra slime analysis kit in xenobio *no changelog needed, but note needed*

PestoVerde322 commented:
Issued Version 1.7.1, which fixes the shuttle itself being misaligned and the dock for the shuttle being weird, now in theory all the ships should dock properly without issues.

SergeiKoralev commented:
![image](https://user-images.githubusercontent.com/43328793/144162892-f06aa8c5-c9c9-4ae0-a397-a9e0ad697478.png)
This airlock near chapel has no turf under it.

SergeiKoralev commented:
- [ ] The port quarter solar dual port air vent in the airlock is backwards. it feeds from scrubbers. 
- [ ] Also the door to port quarter solar is mislabeled.
- [ ] I'm all for high security doors being shocked, but at least electrify the windows next to those doors too! (See the bridge main entrance).
- [ ] Monastary maint is still not connected to power at round start.
![image](https://user-images.githubusercontent.com/43328793/144165662-d7a79895-6cff-4a8b-a8f7-d0bfa6097c85.png)
- [ ] Med maint APC does not connect to grid in round start configuration. the catwalk might be interfering.
![image](https://user-images.githubusercontent.com/43328793/144166554-3bbd0906-cc64-41a0-84a8-2c416fa13ede.png)


Also I feel this should be a given, but there are wall pipes everywhere.

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Updating Code as per usual.

PestoVerde322 committed:
Flandstation 1.8

Added new escape pods, added new random rooms, Fixed some stuff left and right *see changelog*

PestoVerde322 commented:
Hey all, V1.8 is finally out!

Here's the changelog:
- Added two new pod docks in accordance to the more pods PR from St0rmC4st3r (https://github.com/BeeStation/BeeStation-Hornet/pull/5827), one by the bridge, the other by the Arcade.
- Made the escape pod below the turbine more reachable for Scientists.
- Added 4 more random rooms to the random room in front of the security.
- Removed some Bee related issues to one of those said rooms.
- Tweaked a bit some ends around the station to indicate that in fact, it's a dead end.
- The engine should be easier to set up now.
- Given the Engineering SMES a little bit more juice to not run out of steam in 10 minutes flat.
- Adjusted some graphical issues.
- The cargo ship should now have tiny vents.
- Moved some stuff around to places that makes more sense (or to not be in the way of other stuff)
- Harmonized Lights switches, buttons and fire extinguisher closets, as well Request consoles.
- Adjusted some AACs.
- Other stuff i likeley forgot.

Stay tuned for more changes.

PestoVerde322 committed:
Flandstation 1.8.1

Fixes some stuff  *see changelog*

PestoVerde322 commented:
Usual post big release fix:
- Fixed the E.X.P.E.R.I.M.A.N.T.O.R, finally it connects right.
- Plugged a hole in the exper. room.
- Added a Sink in the science janitorial closet.
- Removed the Tritium cans in the engine room.
- Wired the Port Quarter Solars APC properly.

SergeiKoralev commented:
Alright, finally pulled the branch and taking a look at it. 

Areas which remain without an air alarm
- [ ] /area/maintenance/port
- [ ] /area/maintenance/department/chapel/monastery
- [ ] /area/maintenance/port/fore
- [ ] /area/docking/arrivalaux
- [ ] /area/hallway/secondary/service
- [ ] /area/hallway/secondary/exit
- [ ] /area/maintenance/port/aft
- [ ] /area/maintenance/department/medical
- [ ] /area/maintenance/central
- [ ] /area/maintenance/department/security/brig
- [ ] /area/maintenance/port/central
- [ ] /area/maintenance/solars/port/fore
- [ ] /area/maintenance/aft
- [ ] /area/maintenance/starboard/central
- [ ] /area/storage/tech
- [ ] /area/maintenance/starboard/aft
- [ ] /area/docking/bridge
- [ ] /area/maintenance/starboard/fore

A bunch of pipe errors. (Debug verbs - enable > Mapping > check plumbing)
- [ ] Unconnected Vacant Food Stall Commissary vent pump # 1 located at Vacant Food Stall Commissary (139, 165, 2) (JMP)
- [ ] Unconnected scrubbers pipe located at Chapel Office (46, 187, 2) (JMP)
- [ ] Unconnected scrubbers pipe located at Port Solar Maintenance (49, 131, 2) (JMP)
- [ ] Unconnected pipe located at Arrival Shuttle Hallway (57, 120, 2) (JMP)
- [ ] Unconnected air supply pipe located at Port Maintenance (58, 141, 2) (JMP)
- [ ] Unconnected pipe located at Arrival Shuttle Hallway (76, 101, 2) (JMP)
- [ ] Unconnected air supply pipe located at Research Division (132, 110, 2) (JMP)
- [ ] Unconnected scrubbers pipe located at Research Division (132, 110, 2) (JMP)
- [ ] Unconnected scrubbers pipe located at Recreation Area (156, 130, 2) (JMP)
- [ ] Unconnected air supply pipe located at Recreation Area (156, 130, 2) (JMP)
- [ ] Unconnected scrubbers pipe located at Outer Asteroid Prison Wing Arrival (218, 42, 2) (JMP)
- [ ] Errorneous connections around scrubbers pipe. Duplicate or rogue pipes suspected at or around Arrival Shuttle Hallway (50, 102, 2) (JMP) x2
- [ ] Errorneous connections around air supply pipe. Duplicate or rogue pipes suspected at or around Engineering Maintenance (191, 89, 2) (JMP) x2
- [ ] Errorneous connections around scrubbers pipe. Duplicate or rogue pipes suspected at or around Engineering Maintenance (202, 106, 2) (JMP) x2
- [ ] Errorneous connections around pipe manifold. Duplicate or rogue pipes suspected at or around Exploration Shuttle (106, 61, 2) (JMP) x6
- [ ] Errorneous connections around scrubbers pipe. Duplicate or rogue pipes suspected at or around Unique Commissary (132, 111, 2) (JMP)
- [ ] Errorneous connections around air supply pipe. Duplicate or rogue pipes suspected at or around Unique Commissary (132, 111, 2) (JMP)

- [ ] Broken airlock below detective's office. 
![image](https://user-images.githubusercontent.com/43328793/144777090-bcb7728a-15c0-431f-aba6-9ba04ce685f1.png)

- [ ] Air alarm in the arrivals dock is the wrong type. 
![image](https://user-images.githubusercontent.com/43328793/144777179-3981954b-f98e-46b2-baa2-12a47faab5fc.png)

Crossedfall reviewed (dismissed):
Remove the "flat" tiles, they don't fit in with the rest of the art style. Also remove the bad text overlays in atmos. It's a cute reference, but they just don't look good.

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Update to get that sweet sweet overlay lighting.

PestoVerde322 commented:
> Remove the "flat" tiles, they don't fit in with the rest of the art style. Also remove the bad text overlays in atmos. It's a cute reference, but they just don't look good.

I can remove the decals, however, i'm not 100% sure on which flat tiles you're referring to.

Edit: nvm, figured it out which one.

PestoVerde322 committed:
Flandstation 1.8.2

Addresses some fixes and whatnot, Remove Fulltiles, remove Atmos Decals *changelog needed*

PestoVerde322 commented:
Update time:
- Removed the Decals In atmos.
- Removed all the instances of fulltiles.
- Added several air alarms trought the station, aswell light switches. 
- Fixed the broken airlock below detective's office.
- Fixed many other issues.

PestoVerde322 committed:
Flandstation 1.8.2 fix try n.1

Something somewhere broke

PestoVerde322 committed:
Flandstation 1.8.2 fix n.2

Removed the air sensors in the shuttle docking areas, hopefully this won't break the git again.

KubeRoot commented:
Duplicate disposals pipe at (118, 195):
![image](https://user-images.githubusercontent.com/6917698/145039793-4382632d-c5ff-40be-b886-ba521ba6e303.png)

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Update to thing, adding those Server Door access and the Engineering auto-meta-whateverlathe

PestoVerde322 committed:
Flandstation 1.8.3

Add the new Circuit printer thing, Add Proper Server access, add some drapes *NO changelog needed*

PestoVerde322 committed:
Flandstation 1.8.3.1

Fixed an airleak and wrong access to the server room

Crossedfall commented:
Forcing re-lint

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStation
Branch update.

PestoVerde322 committed:
Flandstation 1.9

Update to the structure of the station*see changelog*

PestoVerde322 committed:
Flandstation 1.9 - Exploshuttle fix

Forgot to update the exploration shuttle to the new Tile version

PestoVerde322 commented:
hey everyone, V1.9 is out:

-Moved EVA to where Construction was for an ease of access to the EVA tools, also being more secure in case of breaches.
-Moved Construction where the gravity engine/exploration office was, now the exploration office not only will have room for expansion, but also it's entrance is right in front of their shuttle.
Moved the gravity engine to where EVA was, just looks better and more logical there.
-Fixed the floating switches in the library.
-Moved the Xmas tree to the bar
-Added a new kind of coloured tile by the bridge.
-Technical: made the compressed tile decoration to be under their respective colour tabs.
-Fixed other small stuff.
-The access from the engineering corridor to the engineering shared storage is now possible as an engineer, no more walking trough the SM room.
-Added a teleporter beacon in the permaroid.
-Other stuff i forgot.

Stay tuned for more changes!

PestoVerde322 committed:
Flandstation 1.9.1

Fixed a missing holiday tree and a door in cargo not openable by miners, that's it.

PestoVerde322 commented:
I just updated for a missing holiday tree, also removed a broken stuff and fixed that cargo door.

Mails-z commented:
The vault shouden't have a maintenance entrance here, since the station is so large the silo is not very vigilante and need compensation.

PestoVerde322 commented:
> Ore silo shouden't have a maintenance entrance here, since the station is so large the silo is not very vigilante and need compensation.

I'm sorry, but the Ore silo is safely located into the Vault, are you sure you aren't mistaking it for the Ore Redemption Machine?

KubeRoot commented:
Are those changes to floor sprites intended?
![image](https://user-images.githubusercontent.com/6917698/147374367-7caa4abb-4812-4fdc-908b-5b8bf57797ce.png)

They don't seem documented in the PR body or the changelog, and I don't know if it's currently used.

PestoVerde322 commented:
> Are those changes to floor sprites intended? ![image](https://user-images.githubusercontent.com/6917698/147374367-7caa4abb-4812-4fdc-908b-5b8bf57797ce.png)
> 
> They don't seem documented in the PR body or the changelog, and I don't know if it's currently used.

Forgot to document, but yeah it should be a parallel to the "default" dark version.

KubeRoot commented:
> it should be a parallel to the "default" dark version.

If I understood correctly, your PR is *changing the default* rather than introducing a new version?

PestoVerde322 committed:
Flandstation 1.9.2

Some more small urgent fixes related to atmospherics and a broken door in cargo *no changelog needed*

PestoVerde322 committed:
Flandstation 1.9.2.1

Urgent fixes to the engine, broken pipings and shutters *no changelog required*

Mails-z commented:
![DISCONECTED](https://user-images.githubusercontent.com/72167130/147467547-f241eb57-40dc-476e-9571-21dcf141595d.png)
I found why the bridge go always out of power, the cable of the IPC is 1 block up, in this image I add a new cable

PestoVerde322 committed:
Flandstation 1.10

Added indicators all around, station maps, renewed Cargo, Renewed Atmos *see changelog*

Archanial commented on config/maps.txt:
```diff
@@ -49,6 +49,12 @@ map kilostation
 
+map flandstation
+	minplayers 45
+	maxplayers 120
```
```suggestion
	minplayers 40
```

Archanial committed:
Update config/maps.txt
